src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ManageOffencesApiClient.kt

```diff
@@ -16,33 +18,42 @@ class ManageOffencesApiClient(@Qualifier("manageOffencesApiWebClient") private v
 
   fun getPCSCMarkersForOffences(offenceCodes: List<String>): List<OffencePcscMarkers> {
     val offencesList = if (offenceCodes.size > 1) offenceCodes.joinToString(",") else offenceCodes[0]
-    log.info("/schedule/pcsc-indicators?offenceCodes=$offencesList")
+    log.info("getPCSCMarkersForOffences : /schedule/pcsc-indicators?offenceCodes=$offencesList")
 
     return webClient.get()
       .uri("/schedule/pcsc-indicators?offenceCodes=$offencesList")
       .retrieve()
       .bodyToMono(typeReference<List<OffencePcscMarkers>>())
-      .onErrorMap {
-        CouldNotGetMoOffenceInformation(
-          "PCSC indicator for offence lookup failed for $offencesList," +
-            " can not proceed to perform a sentence calculation",
-        )
+      .retryWhen(
+        Retry.backoff(5, Duration.ofMillis(100))
+          .maxBackoff(Duration.ofSeconds(3))
+          .doBeforeRetry { retrySignal ->
+            log.warn("getPCSCMarkersForOffences: Retrying [Attempt: ${retrySignal.totalRetries() + 1}] due to ${retrySignal.failure().message}. ")
+          },
+      )
```

Comment:
Could potentially filter this down a bit rather than ALL errors but it's good for now